### PR TITLE
Combobox options expand on tab

### PR DESCRIPTION
**Pull request by UnknownSean8**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description
Fixes Issue #618. Tabbing into the combo box will now have the same functionality as clicking into it. The second commit is just adding comments.

### Related issue

- #618

### CODE REVIEW

File: frontend/components/combobox/ComboboxTopics.vue [hunk 1]
```diff
@@ -29,6 +30,7 @@
         >
           <ComboboxOptions
             class="absolute mt-1 max-h-60 w-full overflow-auto rounded-md bg-light-distinct dark:bg-dark-distinct py-1 text-base elem-shadow-lg ring-1 ring-black/5 focus:outline-none sm:text-sm"
+            id="isVisibleElement"
```
Comment (andrewtavis) [inline frontend/components/combobox/ComboboxTopics.vue:33]: We just need the `id` to go above class here given the [eslint order](https://github.com/activist-org/activist/blob/ff0f8b0bfcf2f665236c7f3e5480e12f0006a307/frontend/.eslintrc.js#L40), @UnknownSean8 :) Also happy to do this myself!

**Comment by andrewtavis** [comment]
Haha you were too fast! Thanks @UnknownSean8!

**Comment by UnknownSean8** [comment]
Haha, I haven't actually read your comment just now, but I forgot to do the formatting.

**Review by andrewtavis — APPROVED** [review]
Functionally really wonderful to use and code well very well implemented ✅ Thanks so much for this, @UnknownSean8! Really a pleasure to communicate and work with you on this 😊 Hope you contribute again!

**Comment by UnknownSean8** [comment]
Thank you as well, was looking for a project to improve on my VueJS skills and I believe I can learn a lot by contributing to your project.

I will contribute to other issues that I see fit soon. It was a pleasure to communicate with you as well.

**Pull request merged**